x-pack/legacy/plugins/siem/server/lib/timeline/create_timelines_stream_from_ndjson.ts

```diff
@@ -5,6 +5,10 @@
  */
 
 import { Transform } from 'stream';
+import { identity } from 'fp-ts/lib/function';
+import Boom from 'boom';
```

Comment:
We've removed most (all?) references to Boom from siem, opting instead to use the `KibanaResponseFactory` if it's a simple response, and our `SiemResponseFactory` if it needs a custom message.

I don't have a strong preference for one pattern or the other, but I think we should try to consolidate to one approach. Given the existence of [`router.handleLegacyErrors`](https://github.com/elastic/kibana/blob/master/src/core/MIGRATION_EXAMPLES.md#3-new-platform-shim-using-new-platform-router), though, I do think that the solution should remove any direct dependencies on Boom.

Reply:
Thanks for pointing this out. While I was moving away from Boom, found some applicable library. It uses the simplest `Error` and it seems it's doing its job.

Reply:
@angorayc no problem! I'm happy to do a review when this is ready to go, just let me know.